TypeSystem/Schema/smoke_test_struct.h

```diff
@@ -20,6 +21,7 @@ CURRENT_STRUCT(Primitives) {
   CURRENT_FIELD(m, bool);
   CURRENT_FIELD(n, std::chrono::microseconds);
   CURRENT_FIELD(o, std::chrono::milliseconds);
+  CURRENT_FIELD_DESCRIPTION(b, "Field descriptions can be set in any order.");
```

Comment:
Great idea! Doing now. Nit: the `microseconds` and `milliseconds` are not user-level comments but type comments (annotations for `int64` as their C++ types are `std::chrono::{micro/milli}seconds` respectively). They would stay the way it is right now.